recipe-server/client/control_new/components/recipes/PreferenceExperimentFields.js

```diff
@@ -51,13 +52,13 @@ export default class PreferenceExperimentFields extends React.Component {
           </FormItem>
         </Col>
         <Col sm={24} md={{ span: 12, offset: 1 }}>
-          <FormItem
+          <TrimWhitespaceFormItem
```

Comment:
I think I'd prefer turning the whitespace trim into a prop on our already-custom `FormItem` over using a new component. To me, it's cleaner to use one component and write `<FormItem trimWhitespace ... />`. This could be a personal preference though - @mythmon @rehandalal @Osmose  any thoughts?

Reply:
+1 to a prop instead of a separate component.